Code2Gether-Discord-Bot.Tests/TestConfig.cs

```diff
@@ -0,0 +1,215 @@
+    internal static class TestConfig
+    {
+        #region BusinessLogicFactory
+        /// <summary>
+        /// Instantiates a generic <see cref="Library.BusinessLogic.ExcuseGeneratorLogic"/>.
+        /// </summary>
+        /// <returns>ExcuseGeneratorLogic with irrelevant properties.</returns>
+        public static IBusinessLogic ExcuseGeneratorLogic()
+        {
+            var classContext = typeof(object);
+            var context = CommandContext();
+
+            return BusinessLogicFactory.ExcuseGeneratorLogic(classContext, context);
+        }
+        /// <summary>
+        /// Instantiates a generic <see cref="Library.BusinessLogic.InfoLogic"/>.
+        /// </summary>
+        /// <returns>InfoLogic with irrelevant properties.</returns>
+        public static IBusinessLogic InfoLogic()
+        {
+            var classContext = typeof(object);
+            var context = CommandContext();
+
+            return BusinessLogicFactory.GetInfoLogic(classContext, context);
+        }
+        /// <summary>
+        /// Instantiates a generic <see cref="Library.BusinessLogic.JoinProjectLogic"/> with a custom project repository.
+        /// </summary>
+        /// <param name="projectRepository">This instance's project repository.</param>
+        /// <param name="projectName">The name of the project to join.</param>
+        /// <returns>JoinProjectLogic with a custom project repository and irrelevant properties.</returns>
+        public static IBusinessLogic JoinProjectLogic(IProjectRepository projectRepository, string projectName)
+        {
+            var context = CommandContext();
+
+            return new JoinProjectLogic(Logger(), context, ProjectManager(projectRepository), projectName);
+        }
+        /// <summary>
+        /// Instantiates a generic <see cref="Library.BusinessLogic.ListProjectsLogic"/> with a custom project repository.
+        /// </summary>
+        /// <param name="projectRepository">This instance's project repository.</param>
+        /// <returns>ListProjectsLogic with a custom project repository and irrelevant properties.</returns>
+        public static IBusinessLogic ListProjectsLogic(IProjectRepository projectRepository)
+        {
+            var context = CommandContext();
+
+            return new ListProjectsLogic(Logger(), context, projectRepository);
+        }
+        /// <summary>
+        /// Instantiates a generic <see cref="Library.BusinessLogic.MakeChannelLogic"/>.
+        /// </summary>
+        /// <returns>MakeChannelLogic with irrelevant properties.</returns>
+        public static IBusinessLogic MakeChannelLogic()
+        {
+            var classContext = typeof(object);
+            var context = CommandContext();
+            context.Message = UserMessage(context.Message.Author as FakeUser, "debug!makechannel make-me");
+            var channelName = "test-channel";
+
+            return BusinessLogicFactory.GetMakeChannelLogic(classContext, context, channelName);
```

Comment:
Factory should not be used, as it is the production implementation. During testing `new MakechannelLogic` can be used.

Reply:
Fixed.